src/interface/compare.jl

```diff
@@ -55,3 +55,69 @@ end
 function Base.isequal(A::AbstractArray, B::AbstractTensor)
     return helper_isequal(A, B)
 end
+
+function helper_argmin(A, dims)
+    @assert 1 <= dims <= length(size(A))
+    if (length(size(A)) >= 2)
+        return map(x -> x[2], reduce(minby, map(Pair, A, CartesianIndices(size(A))), dims=dims, init=Inf=>CartesianIndex(fill(0, length(size(A)))...)))
+    else
+        return map(x -> x[2], reduce(minby, map(Pair, A, 1:size(A)[1]), dims=dims, init=Inf=>0))
+    end
+end
+
+function helper_argmin(A)
```

Comment:
could this be defined in terms of the previous function? i.e. `helper_argmin(A) = helper_argmin(A, (1:ndims(A)...,))`?